src/Notifications/AbstractNotification.php

```diff
@@ -63,4 +72,92 @@ abstract class AbstractNotification
 
         return config($config_key, []);
     }
+
+    /**
+     * Return the class of a specific provider implementation.
+     *
+     * @param string $provider
+     *
+     * @return string
+     * @throws \Herpaderpaldent\Seat\SeatNotifications\Exceptions\UnknownDriverException
+     * @example
+     * ```
+     *   ProviderANotificationImplementationClass
+     * ```
+     */
+    final public static function getDiverImplementation(string $provider): string
+    {
+        try {
+            // build the configuration key related to the notification
+            $config_key = sprintf('services.seat-notification.%s.%s', get_called_class(), $provider);
+
+            return config($config_key, []);
+        } catch (\Throwable $exception) {
+
+            // throw exception if implementation is not found.
+            throw new UnknownDriverException($provider);
```

Comment:
It must be always throw if the requested driver is not found which will not be the case actually.

You'll have to check the return of `config`

Reply:
Yep implemented as such: https://github.com/herpaderpaldent/seat-notifications/pull/42/commits/d3544d3f74b8467ec886ef3a2f17df9f0fad2ddd